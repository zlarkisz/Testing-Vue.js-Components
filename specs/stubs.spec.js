import { shallowMount } from "@vue/test-utils";

import ListComponent from "@/list.vue";

const ListItemStub = {
  template: `<li>{{ movie }}</li>`,
  props: ["movie"],
};

test("shallowMount", () => {
  const wrapper = shallowMount(ListComponent, {
    stubs: {
      ListItem: ListItemStub,
    },
  });
  expect(wrapper).toMatchSnapshot();
});
