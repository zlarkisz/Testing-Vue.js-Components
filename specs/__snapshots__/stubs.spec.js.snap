// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallowMount 1`] = `
<div>
  <ul>
    <li>Iron Man</li>
    <li>Avengers</li>
    <li>Infitinity War</li>
  </ul>
</div>
`;
